src/components/Card/ProductCard.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Button from '../Button/Button';
+import './ProductCard.css';
+
+const ProductCard = ({ product, addToCart }) => {
+  const handleAddToCart = () => {
+    if (typeof addToCart === 'function') {
+      addToCart();
```

Comment:
You don't pass amount of selected items to this function. It looks you are not able to select more then 1 item to add to cart